src/qml/CoordinateLocator.qml

```diff
@@ -200,6 +235,40 @@ Item {
     }
   }
 
+  Shape {
+    id: snapToCommonAngleLines
+
+    property double endCoordX: 0
+    property double endCoordY: 0
+
+    visible: !!locator.commonAngleInDegrees || ( endCoordX !== 0 && endCoordY !== 0 )
+    width: parent.width
+    height: parent.height
+    anchors.centerIn: parent
+
+    // outer line
+    ShapePath {
+      strokeWidth: 4
```

Comment:
To avoid this outer line value getting out of sync, I'd suggest we give an id to the inner line, and have the strokeWidth here be .e.g innerLine.strokeWidth + 2.

Reply:
Done.